doc/tutorial/fcm_specifics

```diff
@@ -0,0 +1,1484 @@
+                                <li class="dropdown-header">Quizzes</li>
+                                <li><a href="fundamentals_quiz">Fundamentals Quiz</a></li>
+                                <li><a href="essentials_quiz">Essentials Quiz</a></li>
+                                <li><a href="specifics_quiz">Specifics Quiz</a></li>
+                                <li role="separator" class="divider"></li>
+                                <li class="dropdown-header">Help/info</li>
+                                <li><a href="fcm_glossary">Glossary List</a></li>
+                                <li><a href="fcm_links">Links</a></li>
+                                <li><a href="fcm_mqrg">My Quick Reference Guide</a></li>
+                            </ul>
+                        </li>
+                    </ul>
+                </div>
+                <!--/.nav-collapse -->
+            </div>
+        </nav>
+        <div class="container theme-showcase" role="main">
+            <div class="jumbotron">
+                <h1>Specifics of fcm</h1>
+                <p>This aims to explain and provide tutorial examples for the following topics:
+                <ol>
+                    Repository setup
+                    <ul>
+                        <li><a href="#S1">Setting-up the tutorial repositories</a></li>
+                    </ul>
+                    Specifics
+                    <ul>
+                        <li><a href="#S2">More on merge</a></li>
+                        <li><a href="#S3">propset/del/edit</a></li>
+                        <li><a href="#S4">Trac, browse, branch-diff -t, tickets</a></li>
+                        <li><a href="#S5">Text conflicts, conflicts, resolve, tree conflicts</a></li>
+                        <li><a href="#S6">fcm make, cherry pick</a></li>
+                    </ul>
+                </ol>
+                </p>
+            </div>
+            <div class="well">
+                Now lets do the Specificss Quiz if you have not already done it at the end of  
+                the Essentials section to see how much you already know and what we will need 
+                to focus on in this section.
+            </div>
+            <div class="container">
+                <p><a class="btn btn-lg btn-success" href="specifics_quiz" role="button">Specifics Quiz</a></p>
+            </div>
+            <br/>
+            <h1>Repository setup</h1>
+            <div class="page-header">
+                <h2><a id="S1">Setting-up the tutorial repositories</a></h2>
+            </div>
+            <div class="well">
+                Please run the following <a href="./repos/tutorial_repo_setup">tutorial repositories script</a> if you have not already done so by running the following 
+                command in a terminal 
+                <code>~/kerryday/fcm_tutorial_docs/bootstrap/docs/repos/tutorial_repo_setup</code> this will set-up the repositories that you will need for all of the tutorials.
```

Comment:
Presumably this should just be a call to the command rather than link to your space